src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapper.java

```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.InstantType;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.UriType;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.PathologyRecord;
+
+import java.util.List;
+
+@Component
+@Slf4j
+public class BundleMapper {
+    private static final List<UriType> BUNDLE_META_PROFILE = List.of(new UriType("https://fhir.nhs.uk/STU3/StructureDefinition/ITK-Message-Bundle-1"));
+    private static final String BUNDLE_IDENTIFIER_SYSTEM = "https://tools.ietf.org/html/rfc4122";
+
+    public Bundle mapToBundle(PathologyRecord pathologyRecord) {
+        Bundle bundle = generateInitialPathologyBundle();
+
+        // TODO: fullUrl is optional. But what is the fullUrl property on the entry? E.g., "urn:uuid:a4409d7c-b613-477c-b623-87e60406c2f0"
```

Comment:
Do we have a ticket in the backlog for the TODO's in this class which we can add to the comment? Just so that they're not missed/ignored

Reply:
Removed those TODOs for now. I've asked Philip these questions but he hasn't got back to me yet. I've generated UUIDs for them for now, but these will need addressed in a later story